src/code/utils/js-plumb-diagram-toolkit.coffee

```diff
@@ -226,11 +230,16 @@ module.exports = class DiagramToolkit
     if opts.showIndicators? and not opts.showIndicators
       changeIndicator = null
 
+    if opts.isTransfer
+      @kit.importDefaults
+        Connector: ["Flowchart", {}]
+        Anchor:    ["Continuous", { faces:["left","right"] }]
+
```

Comment:
I'm played around with it and it didn't seem to make any difference, but there is another call to `@kit.importDefaults()` at the end of the function which seems to serve the purpose of resetting the defaults, so I added an `Anchor` to that call for good measure.